hadoop-ozone/dist/src/main/compose/testlib.sh

```diff
@@ -82,9 +82,14 @@ start_docker_env(){
   local -i datanode_count=${1:-3}
 
   docker-compose -f "$COMPOSE_FILE" down
-  docker-compose -f "$COMPOSE_FILE" up -d --scale datanode="${datanode_count}"
-  wait_for_datanodes "$COMPOSE_FILE" "${datanode_count}"
-  sleep 10
+  docker-compose -f "$COMPOSE_FILE" up -d --scale datanode="${datanode_count}" \
+    && wait_for_datanodes "$COMPOSE_FILE" "${datanode_count}" \
+    && sleep 10
+
+  if [[ $? -gt 0 ]]; then
```

Comment:
shellcheck:9: note: Check exit code directly with e.g. 'if mycmd;', not indirectly with $?. [SC2181]